README.md

```diff
@@ -100,6 +100,71 @@ Options:
   -h, --help                       output usage information
 ```
 
+### Version 2
```

Comment:
What's the story with "Version 2" and local development? If I'm making changes in `ui-components` repo and want to see those changes in my local monolith instance how does that work?

Reply:
I didn't think I needed to address local development here since that is taken care of by the `ui-components` repo but maybe I still misunderstand how the local environment works with between `ui-components` and the monolith. Its possible I'll need a bit of help here when it comes time but I don't believe anything in this repo is responsible for that... correct me if I'm wrong.

Reply:
I guess a follow up to this is `ui-components` will be able to take care of local development with the new changes I'll be putting up in the webpack config file. Because local dev won't be using the versioned hashes (as far as I understand) the way gurgler handles things is not important. Again, correct me if I'm wrong.

Reply:
Oh, that's a really good point... local development should **not** be the responsibility of gurgler.